docs/storage/retry_timeout.rst

```diff
@@ -12,22 +12,28 @@ Configuring Timeouts
 --------------------
 
 For a number of reasons, methods which invoke API methods may take
-longer than expected or desired.  By default, such methods all time out
-after a default interval, 60.0 seconds.  Rather than blocking your application
-code for that interval, you may choose to configure explicit timeouts
-in your code, using one of three forms:
+longer than expected or desired. By default, such methods are applied a
+default timeout of 60.0 seconds.
 
-- You can pass a single integer or float which functions as the timeout for the
-  entire request. E.g.:
+The python-storage client uses the timeout mechanics of the underlying
+``requests`` HTTP library. The connect timeout is the number of seconds
+to establish a connection to the server. The read timeout is the number
+of seconds the client will wait for the server to send a response.
+In 99.9% of cases, this is the maximum wait time before the server sends
+the first byte.
+
+Rather than blocking your application code for that interval,you may choose to
```

Comment:
I think the "Rather than blocking your application code for that interval" phrase from our previous version is confusing and we should leave it out.

Reply:
good call; updated!